src/Mod/Draft/draftgeoutils/general.py

```diff
@@ -57,12 +57,16 @@ def precision():
     return precisionInt  # return PARAMGRP.GetInt("precision", 6)
 
 
-def vec(edge):
-    """Return a vector from an edge or a Part.LineSegment."""
-    # if edge is not straight, you'll get strange results!
-    if isinstance(edge, Part.Shape):
-        return edge.Vertexes[-1].Point.sub(edge.Vertexes[0].Point)
-    elif isinstance(edge, Part.LineSegment):
+def vec(edge, use_orientation = False):
+    """ vec(edge[, use_orientation]) or vec(line): returns a vector from an edge or a Part.LineSegment.
+        If use_orientation is True, it takes into account the edges orientation.
+        If edge is not straight, you'll get strange results! """
```

Comment:
The docstring must not repeat the arguments of the function. There must be a single sentence first, then an empty line, and then the rest of the documentation.
```py
def vec(edge, use_orientation=False):
    """Return a vector from an edge or a Part.LineSegment.

    If use_orientation is True, it takes into account the edges orientation.
    If edge is not straight, you'll get strange results!
    """
```